dags/mozaggregator_mobile.py

```diff
@@ -116,7 +116,7 @@
             "bin/export-avro.sh",
             "moz-fx-data-shared-prod",
             "moz-fx-data-shared-prod:analysis",
-            "gs://moz-fx-data-derived-datasets-parquet-tmp/avro/mozaggregator",
+            "gs://moz-fx-data-derived-datasets-parquet-tmp/avro/mozaggregator/mobile",
```

Comment:
Shouldn't this also be included in Dataproc task parameter? (https://github.com/mozilla/telemetry-airflow/blob/9ff864b68240bef41900b1c59114a26fc2c8b45f/dags/mozaggregator_mobile.py#L99)

Reply:
Yes, thank you for catching this.